README.md

```diff
@@ -1 +1,41 @@
-# ToDo-List
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Awesome-Books-ES6
+
+> ### In this project, I built a simple todo list App.
```

Comment:
- [ ] It would be better if you give more information about your project in the `description` section. By doing this you would help the readers a lot.